rusk-wallet/src/lib/store.rs

```diff
@@ -14,6 +14,12 @@ use crate::lib::crypto::{decrypt, encrypt};
 use crate::lib::SEED_SIZE;
 use crate::Error;
 
+/// Specifies the encoding used to save files
+///
+/// Version info is just the first byte, the following two zero's
+/// are used as separators.
+const WALLET_FILE_VERSION: &[u8] = &[1, 0, 0];
```

Comment:
This is limiting us to have only 8 version, right?

Reply:
We can have up to 254 versions, I felt that was enough.